Correct prefixes from -webkit- to webkit in api/ files.

jpmedley opened the pull request:
Pursuant to #3670.

ddbeck commented:
Actually, I think I jumped to conclusions here. Checking on something first.

ddbeck commented on api/MediaSource.json:
```diff
@@ -11,7 +11,7 @@
               "version_added": "23",
               "version_removed": "31",
-              "prefix": "-webkit-"
+              "prefix": "webkit"
```
Spot checking this raised a couple of questions, because I found stuff like this: https://bugs.chromium.org/p/chromium/issues/detail?id=239485

1. I'm wondering if the top-level feature and the constructor prefixes should be `WebKit` instead of `webkit`. Seems like the capitalized version is preferred for those

jpmedley replied:
FYI. This is an exception to how prefixes was/are spelled in Chromium.

ddbeck commented on api/MediaSource.json:
```diff
@@ -154,7 +154,7 @@
                 "version_added": "23",
                 "version_removed": "31",
-                "prefix": "-webkit-"
+                "prefix": "webkit"
```
2. I'm wondering if the prefixes on methods is spurious. In other words, is this really, `webkitMediaSource.webkitSourceBuffers` or is it `webkitMediaSource.sourceBuffers`? That bug link implies the latter, but I'm not quite sure

(Both of these questions apply throughout the PR)

jpmedley commented:
* Fixed the media source APIs.
* `HTMLMediaElement.preservesPitch` is [spelled as in the PR](https://chromium.googlesource.com/chromium/src/+/fca4e7ec60a824062589c7a3dde726cb1ccef902/third_party/WebKit/Source/core/html/HTMLMediaElement.idl).
* There are a few places I changed `chrome_android` to 33. The [evidence is here](https://blog.chromium.org/2014/01/chrome-33-beta-custom-elements-web.html).
* `Navigator.webkitGamepads` [is correct](https://chromium.googlesource.com/chromium/src/+/86fb635fe2e0266af88bed41c1b3987fa9b926d4%5E%21/#F15).

I've so far been unable to find any information on Notification. 

FYI, these files have other problems which I'll be addressing in other commits.

jpmedley commented:
PTAL
